src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,11 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+    const array = number.toString().split('').reverse()
+    const array2 = []
+    for (let i=0; i<array.length; i++){
+        array2.push(Number(array[i]))
+    }
+    return array2
+};
```

Comment:
You could do this without the extra variable creation just by chaining a `.map` method on the end of line 2:
```
const numberToReversedDigits = (number) =>
  number
    .toString()
    .split("")
    .reverse()
    .map((num) => Number(num));
```